index.bs

```diff
@@ -5417,15 +5417,15 @@ same [=[RP]=] and passing through the same attacker.
 
 Such an attack would potentially be detectable; since the [=[RP]=] has registered the attacker's [=credential public key=] rather
 than the user's, the attacker must tamper with all subsequent [=authentication ceremonies=] with that [=[RP]=]: unscathed
-ceremonies will fail.
+ceremonies will fail, risking to reveal the attack.
```

Comment:
nice, tho:
s/ , risking to reveal / , potentially revealing the attack. /

Reply:
Done, thanks!